server/channels/api4/brand_test.go

```diff
@@ -64,6 +65,76 @@ func TestUploadBrandImage(t *testing.T) {
 	CheckCreatedStatus(t, resp)
 }
 
+func TestUploadBrandImageTwice(t *testing.T) {
```

Comment:
This seems unrelated?

Reply:
As I mentioned in the PR description, I wanted to test that my changes did break the existing business logic. Hence, I added the test `TestUploadBrandImageTwice` to ensure that. The changes in `server/channels/app/brand.go` were a bit more involved than usual.

Reply:
Apologies, I read your PR description after I posted this comment. All good!